packages/widget_toolkit_pin/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [0.0.2-dev.4]
+* Fix biometrics button being displayed when `biometricsLocalDataSource` is not provided
+* Update dependencies 
 ## [0.0.2-dev.3]
```

Comment:
Please, add a new line between dev.4 and dev.3 descriptions

Reply:
Fixed